test/testSamples/MaterialX/MtlxUVStreamTest.usda

```diff
@@ -0,0 +1,107 @@
+
+    def Scope "Looks"
+    {
+        def Material "standardSurface2SG"
+        {
+            token outputs:mtlx:surface.connect = </pCube1/Looks/standardSurface2SG/standardSurface2.outputs:surface>
+
+            def Shader "standardSurface2"
+            {
+                uniform token info:id = "ND_standard_surface_surfaceshader"
+                float inputs:base = 1
+                color3f inputs:base_color.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG.outputs:baseColor>
+                token outputs:surface
+            }
+
+            def NodeGraph "MayaNG_standardSurface2SG"
+            {
+                color3f outputs:baseColor.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/mix1.outputs:out>
+
+				def Shader "mix1"
+				{
+                    uniform token info:id = "ND_mix_color3"
+                    color3f inputs:fg.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/ramp1.outputs:out>
+                    color3f inputs:bg.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/MayaSwizzle_file1_rgb.outputs:out>
+                    float inputs:mix.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/add1.outputs:out>
+                    color3f outputs:out
+				}
+
+				def Shader "add1"
+				{
+                    uniform token info:id = "ND_add_float"
+					float inputs:in1.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/noise1.outputs:out>
+					float inputs:in2 = 0.5
+					float outputs:out
+				}
+				
+				def Shader "noise1"
+				{
+                    uniform token info:id = "ND_noise2d_float"
+                    float inputs:amplitude = 0.5
+					float2 inputs:texcoord.connect = </pCube1/Looks/standardSurface2SG/MayaNG_standardSurface2SG/UV1.outputs:out>
+                    float outputs:out
+				}
+				
+				def Shader "UV1"
+				{
+                    uniform token info:id = "ND_texcoord_vector2"
+					int inputs:index = 1
+                    float2 outputs:out
+				}
+				
+				def Shader "ramp1"
+				{
+                    uniform token info:id = "ND_ramptb_color3"
+                    color3f inputs:valuet = (0.0, 1.0, 0.5)
+                    color3f inputs:valueb = (1.0, 0.5, 0.0)
+                    color3f outputs:out
```

Comment:
Unconnected texcoord input. Will be auto-connected to an auto-added ST_reader node that reads primvar "st".